src/cf-core/usage.md

```diff
@@ -1043,7 +1072,7 @@ _Visit https://github.com/cfpb/cf-forms for advanced form field patterns._
 <label for="checkbox-input">Checkbox label</label>
 ```
 
-### Form Elements
+### Form elements
```

Comment:
Not this line, but can you change the titles in this section to make it easier to get the permalinks? `type="search"` -> search input